src/schema/core.yaml

```diff
@@ -417,7 +416,17 @@ classes:
       A controlled term or class from an ontology
     slots:
       - term
-    # TODO: add fields for ontology, branch
+    todos:
+      - add fields for ontology, branch
+
+  ControlledIdentifiedTermValue:
+    description: A controlled term or class from an ontology, requiring the presence of term with an id
+    notes:
+      - To be used for slots like env_broad_scale
+    is_a: ControlledTermValue
+    slot_usage:
+      term:
+        required: true
```

Comment:
This is a super useful addition. I had one quick question. Is there a reason why we can't enforce `required: true` on `ControlledTermValue`? 

Also it looks like there's additional requests for expanding what `ControlledTermValue` can capture, i.e., ontology, branch, etc.?

Just an approach I had in mind:

```
ControlledValue
  is_a: AttributeValue

ControlledTermValue:
  is_a: ControlledValue
  slots:
    - term
   slot_usage:
     term:
       required: true

ControlledOntologyValue:
  is_a: ControlledValue
  slots:
    - ontology # to be added?
   slot_usage:
     ontology:
       required: true
```

And so on.

Please feel free to override this, I haven't made too many changes to the schema, and this is just a thought I had.

Reply:
Oh right, that makes sense. To start with modeling classes for the MIxS environmental triad. And I completely missed thinking about the existing `ControlledTermValue` records from MongoDB which have `has_raw_value` and no `id`. Enforcing `id` on `ControlledTermValue` would invalidate the existing records from MongoDB, so a new `ControlledIdentifiedTermValue` class would definitely be the way to go, but we should think about the structure for these objects in later versions.